tests/config/bindings/azure/servicebusqueues/bindings.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: servicebusqueue-binding
+  namespace: default
+spec:
+  type: bindings.azure.servicebusqueues
+  version: v1
+  metadata:
+  - name: connectionString
+    value: ${{AzureServiceBusConnectionString}}
+  - name: queueName
+    value: ${{AzureServiceBusQueue}}
```

Comment:
This secret needs to be part of required_secrets in conformance.yaml (line 69)

Reply:
The first one is, the 2nd one I meant to remove but accidentally removed the wrong config... I've made it hard coded as it doesn't feel like it needs to be a secret to me. Will change if you think it should be.

Reply:
I think it should be fine hard-coded here. If we want to we can extract this out as an environment variable in the workflow later.